recipe/conda_build_config.yaml

```diff
@@ -2,4 +2,4 @@ arm_variant_type: # [aarch64]
   - sbsa          # [aarch64]
 c_stdlib_version:  # [linux]
   - 2.28           # [linux and aarch64]
-  - 2.17           # [linux and x86_64]
+  - 2.28           # [linux and x86_64]
```

Comment:
Have set Linux `x86_64` to use GLIBC 2.28 based on our observations above: https://github.com/conda-forge/libcublas-feedstock/pull/33#issuecomment-2613643869